### PR TITLE
Account missing

### PR DESCRIPTION
**Pull request by mcallb**

### CODE REVIEW

**Comment by venth** [comment]
Hi @mcallb,

please resolve merge conflicts (git pull upstream master - on your feature branch). I wonder about jenkins file and aws cloudformations - why do you need them? aws-adfs is a command line tool.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!groovy
```
Comment (venth) [inline Jenkinsfile]: This file is unnecessary. I use travis CI to build and test the project - please take a look at .travis file.

File: aws_adfs/_version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '0.3.3'
+__version__ = '0.3.5'
```
Comment (venth) [inline aws_adfs/_version.py]: I've changed the versioning mechanism from file base to git based - you can omit changes in this file.

File: aws_adfs/login.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import configparser
-
+import requests
```
Comment (venth) [inline aws_adfs/login.py]: requests and json imports are unnecessary, I think. I can't see any change in this file that requires them.

File: aws_adfs/roles_assertion_extractor.py [hunk 4]
```diff
@@ -37,6 +37,21 @@ def extract(html):
     # Note the format of the attribute value is provider_arn, role_arn
     principal_roles = [role for role in aws_roles if ':saml-provider/' in role[0]]
 
+    # File contains a list of AWS account id's and names
+    # map_file = requests.get('https://s3.amazonaws.com/eis-aws-accounts/account_map.json')
+    map_file = requests.get('https://aafn.ebsco.cloud')
```
Comment (venth) [inline aws_adfs/roles_assertion_extractor.py]: I think that's your api gateway supported by the lambda function - I think that there is a flow in the design. First you will be charged for every call (please note, that this tool can be used everywhere around the world), besides that I don't see the need to use lambda to fetch account names. Friendly account names are provided by ADFS server and `aws-adfs` already use them to group roles.

File: lambda/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+Here is some automation around this process.
+    
+# Package the script
+zip package.zip ./return_account_info.py
+
+# Create a new function
+aws lambda create-function \
+    --function-name getAccountList \
+    --runtime python2.7 \
+    --role arn:aws:iam::865492507168:role/RoleAWSADFSListAcounts \
```
Comment (venth) [inline lambda/README.md]: I think, that you placed here (by mistake probably) a real account number ;) Just for future notice, consider use something like 123456789012 instead ;)

File: lambda/README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    --region us-east-1
+  
+# Add permissions for API Gateway to execute the function
+aws lambda add-permission \
+    --region us-east-1 \
+    --function-name getAccountList \
+    --statement-id 1 \
+    --principal apigateway.amazonaws.com \
+    --action lambda:InvokeFunction \
+    --source-arn arn:aws:execute-api:us-east-1:865492507168:<rest-api-id>/*/GET/
```
Comment (venth) [inline lambda/README.md]: same as afore mentioned -  consider to use something like 123456789012 instead

File: lambda/deploy_function.yaml [hunk 7]
```diff
@@ -0,0 +1,29 @@
+AWSTemplateFormatVersion: '2010-09-09'
```
Comment (venth) [inline lambda/deploy_function.yaml]: AWS lambda isn't needed. ADFS server provides human friendly account names. They are already used to group aws roles within an account.

**Comment by venth** [comment]
OK, after review I think I got your idea to supply aws account names via api gateway and aws lambda. ADFS server provides information about AWS account names. The account names are already used to group aws roles to easier choice of a role.

**Comment by mcallb** [comment]
Hello @venth. I inadvertently opened this pr against your repo. I see now the functionality we were looking for (displaying the account alias) is already available. This is a great enhancement! Thank you for the feedback.

**Comment by venth** [comment]
Thanks @mcallb, a helping hand is always appreciated :)

**Pull request closed without merging**